src/tests/class_tests/openms/source/CrossLinksDB_test.cpp

```diff
@@ -47,6 +47,10 @@ struct ResidueModificationOriginCmp
 {
   bool operator() (const ResidueModification* a, const ResidueModification* b)
```

Comment:
That's just for the test (you want to make sure the order is reproducible between systems).